content/en/docs/KRE/user/50_krt_v2/_index.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "KRT V2"
+linkTitle: "KRT V2"
+weight: 50
+description: >
+  Understading how the new version V2 of KRT works
+---
+
+- [Branched workflows](#branched-workflows)
+- [Reactive approach](#reactive-approach)
+- [Exitpoint](#exitpoint)
+- [KRT YAML File](#krt-yaml-file)
+- [Diagram of an architecture defined by KRT V2](#diagram-of-an-architecture-defined-by-krt-v2)
+
+## Branched workflows
+
+The main purpose of KRT V2 is to bring branched workflows on board. This means nodes are now capable
+of subscribing to several other nodes at the same time, thus workflows are now capable of branching out.
+
+It is advised to previously read and fully understand [KRT](40_krt) before moving on to V2, as this page focuses on explaining V2 related changes and new concepts.
+
+## Reactive approach
+
+Nodes will publish their results onto their own subject, is up to other nodes whether to subscribe and process its messages or not. This allows for organic growth as nodes can be developed then hooked up to other nodes freely.
+
+### Multiple responses
+
+Nodes now also have the ability to publish several to none messages from the same request. So now the _handler_ contract should look like this:
+
+- `handler(ctx *kre.HandlerContext, data *any.Any) (proto.Message, error)`
```

Comment:
It would be nice to have a Python example too

Reply:
Agreed, added